lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -32,7 +32,14 @@
   name:
     description:
       - Name of the security group.
-    required: true
+      - One of and only one of I(name) or I(group_id) is required.
```

Comment:
How about just specifying that it only can be used with absent? Or do you want me to enable the possibility to change security group attributes for specific `group_id`?